cpp/tests/utilities/cudf_test_fixtures.h

```diff
@@ -26,6 +27,15 @@
 #define ASSERT_RMM_SUCCEEDED(expr)  ASSERT_EQ(RMM_SUCCESS, expr)
 #define EXPECT_RMM_SUCCEEDED(expr)  EXPECT_EQ(RMM_SUCCESS, expr)
 
+#define ASSERT_CUDF_SUCCEEDED(gdf_error_expression) \
```

Comment:
Should wrap in the standard `do {...} while(0)`

Reply:
Are you sure? there's no if here.

Reply:
Done.

Reply:
@jrhemstad : Please approve so this can be merged.